beacon_node/execution_layer/src/engine_api.rs

```diff
@@ -531,6 +541,34 @@ impl<E: EthSpec> ExecutionPayloadBodyV1<E> {
                     ))
                 }
             }
+            ExecutionPayloadHeader::Gloas(header) => {
```

Comment:
we removed this Gloas variant from
```
impl<E: EthSpec> ExecutionPayloadBodyV1<E> {
    pub fn to_payload(
        self,
        header: ExecutionPayloadHeader<E>,
    ) -> Result<ExecutionPayload<E>, String> {
```
since there's no longer an `ExecutionPayloadHeader` superstruct variant for Gloas. There are `ExecutionBid` now in beacon blocks.  We shouldn't be using this `to_payload` method anymore for gloas for things like reconstructing blinded blocks since no more blinded blocks. instead, we'll still be using `get_payload` for retrieving locally built payloads and `new_payload` for EL verification of tsx received in `ExecutionPayloadEnvelope`